tests/autorestart/test_container_autorestart.py

```diff
@@ -195,27 +195,27 @@ def kill_process_by_pid(duthost, container_name, program_name, program_pid):
                 .format(program_name, container_name))
 
 
-def is_hiting_start_limit(duthost, container_name):
+def is_hiting_start_limit(duthost, service_name, container_name):
     """
     @summary: Determine whether the container can not be restarted is due to
               start-limit-hit or not
     """
-    service_status = duthost.shell("sudo systemctl status {}.service | grep 'Active'".format(container_name))
+    service_status = duthost.shell("sudo systemctl status {}.service | grep 'Active'".format(service_name))
     for line in service_status["stdout_lines"]:
         if "start-limit-hit" in line:
             return True
 
     return False
 
 
-def clear_failed_flag_and_restart(duthost, service_name):
+def clear_failed_flag_and_restart(duthost, service_name, container_name):
```

Comment:
Same comment as line 198, if we want to use `service_name` to replace `container_name`, we need remove `container_name` from argument list and do the replacement in the code as well for example at line 223.

Reply:
clear_failed_flag_and_restart uses both service_name and container_name, so need to keep both.